services/core/functions/review-lambda/index.ts

```diff
@@ -1,13 +1,14 @@
 import { review } from "../../../../src/review/index";
 import { ReviewArgs, ReviewFile } from "../../../../src/common/types";
+import { APIGatewayProxyEvent } from "aws-lambda";
 import { getVariableFromSSM } from "../helpers";
 
 interface ReviewLambdasBody {
   args: ReviewArgs;
   files: ReviewFile[];
 }
 
```

Comment:
Btw you might want to set the logger level here like I did in the demo lambda, cause otherwise it logs all the debug stuff

Reply:
Ahh! very good point!